### PR TITLE
feat: Feature config APIs

### PR DESCRIPTION
**Pull request by Code0987**

### CODE REVIEW

File: src/spender/openapi.yaml [hunk 1]
```diff
@@ -205,3 +208,7 @@ paths:
     $ref: paths/spender@expenses@check_mandatory_fields.yaml
   /spender/expenses/dismiss_duplicates/bulk:
     $ref: paths/spender@expenses@dismiss_duplicates@bulk.yaml
+  /spender/page_configurations:
```
Comment (aditya999123) [inline src/spender/openapi.yaml:211]: this is not needed

File: src/components/schemas/page_configurations.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      type: string
+      nullable: false
+      description: |
+        This specifies the key of the page configuration.
+      example: COMPANY_EXPENSE_MANAGE_COLUMNS
+    value:
+      $ref: './fields.yaml#/any_type'
+      nullable: false
+      description: |
+        This specifies the value of the page configuration.
```
Comment (aditya999123) [inline src/components/schemas/page_configurations.yaml:25]: page configuration key

File: src/components/schemas/page_configurations.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      $ref: './fields.yaml#/id_integer'
+    key:
+      type: string
+      nullable: false
+      description: |
+        This specifies the key of the page configuration.
+      example: COMPANY_EXPENSE_MANAGE_COLUMNS
+    value:
+      $ref: './fields.yaml#/any_type'
+      nullable: false
```
Comment (aditya999123) [inline src/components/schemas/page_configurations.yaml:54]: nullable true for value

**Review by aditya999123 — APPROVED** [review]

File: src/admin/openapi.yaml [hunk 4]
```diff
@@ -385,4 +388,8 @@ paths:
   /admin/expenses/dismiss_duplicates/bulk:
     $ref: paths/admin@expenses@dismiss_duplicates@bulk.yaml
   /admin/expenses/duplicate_sets:
-    $ref: paths/admin@expenses@duplicate_sets.yaml
\ No newline at end of file
+    $ref: paths/admin@expenses@duplicate_sets.yaml
+  /admin/feature_configs:
```
Comment (aditya999123) [inline src/admin/openapi.yaml:392]: remove this, create only bulk

Comment (Code0987) [reply src/admin/openapi.yaml:392]: This needed for GET api

Comment (Code0987) [reply src/admin/openapi.yaml:392]: Only GET code

**Pull request merged**